packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -128,11 +128,10 @@ const PersonalDetailsForm = props => {
                     'account-form__poi-confirm-example': is_qualified_for_idv,
                 })}
             >
-                {(is_qualified_for_idv || is_rendered_for_onfido) && !should_hide_helper_image && (
-                    <InlineNoteWithIcon
-                        message={name_dob_clarification_message}
-                        font_size={isMobile() ? 'xxxs' : 'xs'}
-                    />
+                {(is_svg_only || is_rendered_for_onfido) && (
```

Comment:
why are we checking landing companies here. Should'nt we check if the account status is verified?

Reply:
This inline message should only be available for svg account creation flow and the idv section in accounts. So to make it work with both, we need to use this condition as `is_qualified_for_idv` flag can be `false` in case of a verified DIEL account